health/REFERENCE.md

```diff
@@ -548,13 +548,13 @@ alert information. Current variables supported are:
 
 | variable | description |
 | ---------| ----------- |
-| $family  | Will be replaced by the family instance for the alert (e.g. eth0) |
-| $label:  | Followed by a chart label name, this will replace the variable with the chart label's value |
+| ${family}  | Will be replaced by the family instance for the alert (e.g. eth0) |
+| ${label:XXX}  | This will replace the variable with the chart label's XXX value |
```

Comment:
```suggestion
| ${label:LABEL_NAME}  | The variable will be replaced with the value of the label |
```

I think we shouldn't use [XXX](https://www.dictionary.com/e/slang/xxx/).

Reply:
Agreed, for multiple reasons.

Reply:
I agree 100% with @Ferroin ! :)

Reply:
Done!!